### PR TITLE
Changes:

### PR DESCRIPTION
**Pull request by EmmanuelAR**
1- Add donator contract
2-Add donatorManager contract
3-Add error constant
4-Add donator constant
5-Add new constants references in all the contracts 6-Add new comments to separete the code in sections

### CODE REVIEW

**Comment by EmmanuelAR** [comment]
TICKETS:
#33
#22
#7

File: contracts/src/constants/donator/donator_constants.cairo [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// *************************************************************************
+//                            DONATOR CONSTANTS
+// *************************************************************************
+pub mod DonatorConstants {
+    pub const INICIAL_LEVEL: u32 = 1;
```
Comment (adrianvrj) [inline contracts/src/constants/donator/donator_constants.cairo:5]: This should be INITIAL

File: contracts/src/constants/donator/donator_constants.cairo [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+    pub const INICIAL_MAX_STARKS_DONATION_TO_NEXT_LEVEL: u64 = 10;
```
Comment (adrianvrj) [inline contracts/src/constants/donator/donator_constants.cairo:6]: Same here

**Review by adrianvrj — CHANGES_REQUESTED** [review]
Couple of orthography changes, everything else looks good

**Review by adrianvrj — APPROVED** [review]

**Pull request merged**